### ISSUE TITLE
Cyclic Enchantment "Multijump" not working with Silent Gear boots

### ISSUE THREAD

**Issue by Flabort** [FTBTeam/FTB-Modpack-Issues#2480]
### Modpack

FTB Presents Direwolf20 1.19

### Modpack version

1.0.4

### Describe the bug

I have some Silent Gear boots. I made them from Electrum, Enchanted them, and upgraded them to Nether Gobber (on a related note, gobber materials lack an Enchantment value for Silent Gear materials). One of the Enchantments I got on my boots is Multi-Jump IV, which claims that it gives a double jump on boots or elytra. Well, I have it on boots, so I expect the Enchantment to function, but it just doesn't. 

### Steps to reproduce

1. Get a boots blueprint, and 4 of the same ingots of almost any metal. Put them in a crafting grid. 
2. Put the boots part in a crafting grid, optionally with any upgrades such as a lining part. 
3. Get a enchanted book with Multi-Jump IV. The Enchantment goes from I to V, but I only tested with IV so far.
4. Apply the Enchantment in an anvil. Alternatively, get lucky and apply it in an Enchanting table.
5. Equip the enchanted Silent Gear boots.
6. Jump, and try to jump in mid-air. 

### Expected behaviour

I expected a double jump as described in the Enchantment description.

**Comment by Saereth** [comment]
https://github.com/SilentChaos512/Silent-Gear/issues/577

**Comment by Bionic711** [comment]
I am not sure this is limited to silent gear boots. I tried with Elementium boots from Botania as well as classic Netherite boots and neither of those worked either. Ranges from IV (classic netherite) to VII (one of three pairs of elementium) (not sure why enchanting is able to make them beyond their "cap" of V). I also tried on a pair of space-boots from Ad Astra with no luck. The multi-jump from Drikwing (not sure if it is the same) in Occultism works fine when equipped on a ring our familiar out in the world, but I hate the slowfall associated with it.
